jupyter_server/services/contents/handlers.py

```diff
@@ -145,16 +145,17 @@ async def get(self, path=""):
             await self._finish_error(
                 HTTPStatus.NOT_FOUND, f"file or directory {path!r} does not exist"
             )
+        kwargs = {
+            "path": path,
+            "type": type,
+            "format": format,
+            "content": content,
+        }
+        if cm.support_md5:
+            kwargs["md5"] = md5
+
```

Comment:
If the `ContentsManager` does not declare itself to support md5(by setting `support_md5` to `True`), `handler` will never pass `md5` argument to the `ContentsManager.get`